demo/Nonlinear Rabbit Population.ipynb

```diff
@@ -0,0 +1,269 @@
+{
```

Comment:
<p>I think this needs to be updated as it is now a standalone demo. There is just a single problem here and hence also check the numbering of below subheader</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/biaslab/ReactiveMP.jl/pull/213/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>